marklogic-data-hub/src/main/java/com/marklogic/hub/impl/MasteringManagerImpl.java

```diff
@@ -114,9 +128,48 @@ public MergeResponse merge(List<String> mergeURIs, String flowName, String stepN
                     resp = null;
                 } else {
                     ResourceServices.ServiceResult res = resultItr.next();
-                    StringHandle handle = new StringHandle();
-                    ObjectMapper objectMapper = new ObjectMapper();
-                    resp = objectMapper.readValue(res.getContent(handle).get(), MergeResponse.class);
+                    JacksonHandle handle = new JacksonHandle();
+                    resp = res.getContent(handle).get();
+                }
+            } catch (Exception e) {
+                throw new RuntimeException(e);
```

Comment:
I don't think this try/catch is needed either, not the use of ServiceResult - should just be able to do:

    return getServices().post(params, jsonOptions, new JacksonHandle()).get();